src/nullablevector.jl

```diff
@@ -311,3 +313,27 @@ function Base.empty!(X::NullableVector)
     empty!(X.isnull)
     return X
 end
+
+function vcat(A::AbstractVector, B::AbstractVector...)
```

Comment:
This is still type piracy. There needs to be `NullableArray` in the signature. Currently I don't think this can be fixed in full generality. We can include a few methods for all possible combinations up to e.g. 3 arguments, i.e:
- `::NullableArray, ::AbstractArray`
- `::AbstractArray, ::NullableArray`
- `::NullableArray, ::AbstractArray, ::AbstractArray`
- `::AbstractArray, ::NullableArray, ::AbstractArray`
- `::AbstractArray, ::AbstractArray, ::NullableArray`